azurelinuxagent/common/event.py

```diff
@@ -237,8 +236,7 @@ def save_event(self, data):
         if len(existing_events) >= 1000:
             # To prevent the chance of #1035 happening - introduced periodic_WARNs.
             # More details about it - azurelinuxagent.common.event.add_log_event's pydoc.
-            logger.periodic_warn(logger.EVERY_FIFTEEN_MINUTES,
-                                 "[PERIODIC] Too many files under: {0}, current count:  {1}, removing oldest".format(
+            logger.warn("Too many files under: {0}, current count:  {1}, removing oldest".format(
```

Comment:
We wont call this function based on your check in Line 305 of this file. Can you move this `logger.warn` to that line?

Reply:
Sorry my bad this is used everywhere. Disregard